src/_blog/webtransport.md

```diff
@@ -0,0 +1,247 @@
+
+Both sides can now open streams (both bidirectional and unidirectional) and send (unreliable) HTTP datagrams.
+
+> Note: the unreliable datagrams are a property of UDP which QUIC is built on top of. This unreliable and out of order delivery is exactly what makes UDP, QUIC, and WebTransport perfect fits for applications like video streaming or browser gaming where low latency and speed are paramount.
+
+In libp2p, we still need to verify the libp2p peer IDs, so we’re not quite done yet.
+
+To do so, the browser opens a new WebTransport stream, and starts a [Noise](https://noiseprotocol.org/noise.html) handshake. This is essentially the same handshake we use to secure connections on top of TCP in libp2p. In this handshake, we bind the outer connection to this handshake, which allows us to verify that we’re not falling victim to a man-in-the-middle (MITM) attack.
+
+The entire process of setting up a WebTransport connection in libp2p therefore takes no more than 3 network roundtrips. Compare that to the 6 roundtrips we needed for WebSocket!
+
+We might even be able to bring this down to 2 roundtrips in the future: In principle, it should be possible to run the `CONNECT` request and the Noise handshake in parallel. Currently, the browser API doesn’t allow this, but we’ve [submitted a proposal](https://github.com/w3c/webtransport/issues/435) to enable this feature.
+
+<!-- TODO: do we need another diagram for this? -->
+
+### Limitations
+
+So far we have discussed connectivity between browsers and servers and how that's enabled by WebTranport and WebSocket. You may have noticed that we have not discussed peer-to-peer browser-to-browser connectivity and another prominent transport: [WebRTC](https://www.w3.org/TR/webrtc/).
+
+This is actually where WebRTC shines and WebTransport falls short: the latter does not support browser-to-browser connectivity. To meet this need, libp2p implementations are in the process of adding support for WebRTC browser-to-server connectivity ([supported in rust-libp2p](https://github.com/libp2p/rust-libp2p/releases/tag/v0.50.0)) as well as browser-to-browser connectivity ([coming soon](https://github.com/libp2p/specs/issues/475).) We plan to share more with you as we make progress on this front.
+
+## What's the current state of WebTransport and where is it supported?
+
+### State of Specifications
+
+### IETF Specs
+
+As we’ve described above, WebTransport is new and as bleeding-edge as it gets.
+
+The IETF specification of the protocol itself is still [in the draft stage with ongoing revisions](https://datatracker.ietf.org/doc/draft-ietf-webtrans-overview/) (the same is true for WebTransport over [HTTP/3](https://datatracker.ietf.org/doc/draft-ietf-webtrans-http3/) and [HTTP/2](https://datatracker.ietf.org/doc/draft-ietf-webtrans-http2/).)
+This will likely be the case for many more months. As such, we don’t know yet how the final version of the protocol will look like.
```

Comment:
```suggestion
This will likely be the case for many more months. As such, we don’t yet know what the final version of the protocol will look like.
```